x/eibc/types/codec.go

```diff
@@ -0,0 +1,24 @@
+package types
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	cdctypes "github.com/cosmos/cosmos-sdk/codec/types"
+
+	// this line is used by starport scaffolding # 1
+	"github.com/cosmos/cosmos-sdk/types/msgservice"
+)
+
+func RegisterCodec(cdc *codec.LegacyAmino) {
+	// this line is used by starport scaffolding # 2
```

Comment:
we need register here Amino types if we want to support eip712 from the portal

Reply:
prob not gonna use it from the portal but gonna register it anyway